vnext/Microsoft.ReactNative.IntegrationTests/pch.h

```diff
@@ -2,8 +2,10 @@
 
 #define NOMINMAX
 
-#include <winrt/Windows.Foundation.Collections.h>
 #include <future>
 #include "NativeModules.h"
 #include "functional/functor.h"
 #include "motifCpp/testCheck.h"
+
+#undef GetCurrentTime
```

Comment:
Nit: push/pop?

Reply:
No, it is going to affect only .cpp files in this project. It gave me compilation errors otherwise.